playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/DigitalGoodsRequestHandler.java

```diff
@@ -0,0 +1,77 @@
+package com.google.androidbrowserhelper.playbilling.digitalgoods;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.RemoteException;
+
+import com.android.billingclient.api.SkuDetails;
+import com.google.androidbrowserhelper.playbilling.provider.BillingWrapper;
+import com.google.androidbrowserhelper.playbilling.provider.BillingWrapperFactory;
+import com.google.androidbrowserhelper.trusted.ExtraCommandHandler;
+
+import java.util.Arrays;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.browser.trusted.TrustedWebActivityCallbackRemote;
+
+public class DigitalGoodsRequestHandler implements ExtraCommandHandler {
+    private final BillingWrapper mWrapper;
+    private final BillingWrapper.Listener mListener = new BillingWrapper.Listener() {
```

Comment:
The mListener seems unused.

Reply:
It's passed as parameter to `BillingWrapperFactory.get()` later - do you mean we don't need it as an instance variable maybe?

Reply:
I meant that it's methods were empty and then it seemed redundant.

Reply:
But I see that it's filled in in the other PR https://github.com/GoogleChrome/android-browser-helper/pull/163/files
so it's ok.